swimster-api/swimster-schema.sql

```diff
@@ -32,6 +32,19 @@ CREATE TABLE listings (
     FOREIGN KEY (host_id) REFERENCES users(id) ON DELETE CASCADE
 );
 
+CREATE TABLE reservations (
+    id          SERIAL PRIMARY KEY,
+    user_id     INT NOT NULL REFERENCES users(id) ON DELETE CASCADE,
+    listing_id  INT NOT NULL REFERENCES listings(id) ON DELETE CASCADE,
+    date        DATE NOT NULL,
+    start_time  TIMESTAMP NOT NULL,
+    end_time    TIMESTAMP NOT NULL,
+    guests      INT NOT NULL,
+    total       BIGINT NOT NULL,
+    status      BOOLEAN NOT NULL,
```

Comment:
Agreed. I wasn't going to focus on this feature until milestone #2 but seems that it would be better to preemptively have it labeled.

Reply:
Are the different state values of type `STRING`? Just wanted to ask before committing the changes

Reply:
Since there's a finite set of possible values, a self-defined type backed by an `ENUM` would be a better choice.

The reference page has some examples on how you can do that: https://www.postgresql.org/docs/current/datatype-enum.html

Reply:
Oh okay, I just wanted to make sure that the identifiers were of type `VARCHAR`. Thanks!